### PR TITLE
updated the ingestion playbook

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
Descriptions:
Updated the ingestion playbook and Get All Indicators action

Mantis Link:
https://mantis.fortinet.com/bug_view_page.php?bug_id=0890060
https://mantis.fortinet.com/bug_view_page.php?bug_id=0890056

UTCs:
Connector installation verified.
Connector logo verified.
Actions and Playbook list verified.
Playbook tags verified.
All Playbook are in info mode verified.
All Playbook are in inactive mode verified.
Ingestion Playbook.
Get All Indicators action.

### CODE REVIEW

File: dragos-worldview-threat-intelligence/operations.py [hunk 1]
```diff
@@ -96,12 +96,36 @@ def build_query(params, query_params):
...
+def get_all_indicators(config, params):
+    fetch_all_records = False
+    if not params.get("page") and not params.get("page_size"):
+        fetch_all_records = True
+    if fetch_all_records:
+        page = 1
+        indicators = list()
+        params.update({"page": page, "page_size": 1000})
+        response = get_indicators_response(config, params)
+        resp_indicators = response.get("indicators", [])
```
Comment (cs-suyog-jinturkar) [inline dragos-worldview-threat-intelligence/operations.py]: I feel, this code has a potential to optimise more.

Comment (cs-omkar-kumbhar) [reply dragos-worldview-threat-intelligence/operations.py]: @cs-suyog-jinturkar this code was tested for 3739 indicators. Can you please suggest how to optimise further ?

Comment (cs-omkar-kumbhar) [reply dragos-worldview-threat-intelligence/operations.py]: @cs-suyog-jinturkar I have done the suggested changes

**Review by cs-suyog-jinturkar — APPROVED** [review]
code changes LGTM

**Pull request merged**